go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/bitcoin-sv/go-broadcast-client
 
-go 1.21.5
+go 1.22
+
+toolchain go1.22.5
 
 require (
 	github.com/libsv/go-bt/v2 v2.2.5
```

Comment:
We still use this somewhere, and I assume that we want to get rid of it also.

Reply:
i got rid of that dependency from go.mod